### PR TITLE
#159250147 Export workouts

### PR DESCRIPTION
**Pull request by archbishop12**
#### What does this PR do?
The application should enable the user to export workouts.

#### Description of Task to be completed?
Exporting workouts as  a json file should be enabled.

#### Description of Task to be completed?
The user should be able to export 

#### How should this be manually tested?
After cloning this repository;
- login into the application
- Click workouts in the navigation bar and then workouts.
- Create a workout.
- After which create the click the options  and click `export as json`
- This download a son file.

#### What are the relevant pivotal tracker stories?
[#159250147](https://www.pivotaltracker.com/story/show/159250147)

#### Screenshots
screen shot 2018-08-29 at 16 43 30
screen shot 2018-08-29 at 14 52 07

### CODE REVIEW

File: wger/manager/views/workout.py [hunk 1]
```diff
@@ -229,6 +231,28 @@ def add(request):
...
+@login_required
+def export_json(request):
+    '''
+    Exports the saved workouts as json data
+    '''
+    workouts = Workout.objects.filter(user_id=request.user.id)
+    workout_id_list = []
+    for workout in workouts:
+        workout_id_list.append(workout.id)
+    list_object = []
```
Comment (JaymesKat) [inline wger/manager/views/workout.py]: This variable name is not very descriptive of the data we are expecting to export. Kindly rename it

**Review by mutebironald — APPROVED** [review]
I have pulled and tested it out.

**Review by JaymesKat — CHANGES_REQUESTED** [review]

File: wger/manager/views/workout.py (hunk 1 above)
Comment (archbishop12) [reply wger/manager/views/workout.py]: Worked on it @Jpkat92

**Review by JaymesKat — APPROVED** [review]

File: wger/manager/models.py [hunk 2]
```diff
@@ -565,6 +565,69 @@ def get_canonical_representation(self):
                 },
                 'set_list': canonical_repr}
 
+    def get_json_representation(self):
```
Comment (isheebo) [inline wger/manager/models.py:568]: nope, break this down my friend into smaller functions/methods. okay @archbishop12? 30 or less lines per function 😄

File: wger/manager/views/workout.py [hunk 3]
```diff
@@ -48,6 +49,7 @@
     WgerDeleteMixin
 )
 from wger.utils.helpers import make_token
+import json
```
Comment (isheebo) [inline wger/manager/views/workout.py]: sort imports here

File: wger/manager/models.py (hunk 2 above)
Comment (archbishop12) [reply wger/manager/models.py:568]: @isheebo I have broken down the function.

File: wger/manager/views/workout.py (hunk 3 above)
Comment (archbishop12) [reply wger/manager/views/workout.py]: @isheebo I have worked on this

**Pull request merged**